site/viamguides/src/scuttle-base/scuttle-base.md

```diff
@@ -0,0 +1,282 @@
+author: Nick Hehr
+id: scuttle-base
+summary: Configure Viam to drive a SCUTTLE base
+categories: Getting-Started, Developer
+environments: web
+status: Published 
+feedback link: https://github.com/viam-labs/viamcodelabs/issues
+tags: Getting Started, Developer, Scuttle, Rover
+
+# Build a SCUTTLE bot
+<!-- ------------------------ -->
+## Overview 
+Duration: 2
+
+Mobile robots come in all shapes and sizes, but most kits come with a pre-defined platform that are tough to change for custom use cases.
+The [SCUTTLE](https://www.scuttlerobot.org/) (Sensing, Connected, Utility Transport Taxi for Level Environments) is a modular, open source robotics platform for building mobile robots that puts you in control with 3D printed parts, extruded aluminum, and DIN railing.
```

Comment:
I know the Scuttle documentation calls the Scuttle robots a "platform" but I would consider adding context like "base" or "chassis" for clarity (when I hear 'platform' I think software)